integration_tests/tsconfig.json

```diff
@@ -3,6 +3,7 @@
     "strict": true,
     "target": "es5",
     "noEmit": true,
+    "skipLibCheck": true,
```

Comment:
Do we need to target `es5` here? No idea if that would fix things but we target `es2018` in the main `tsconfig.json`